frontend/src/components/ClubCard.tsx

```diff
@@ -1,12 +1,14 @@
 import React from "react";
 import { ClubCardContainer, ClubHeaderContainer } from '../styling/StyledHome';
 
-const clubCard = ({application_link, application_name, name, due_date}: Props) => {
+const clubCard = ({application_link, application_name, name, due_date, image}: Props) => {
   return (
     <ClubCardContainer>
       <ClubHeaderContainer>
-        <img src={require("../assets/dti_logo.svg")} alt="logo" />
-        <h3>{name}</h3>
+        {(image !== undefined)? 
```

Comment:
Even if there is an image, don't we still want to display the club name somewhere (so people can Ctrl+F)?